share/completions/brew.fish

```diff
@@ -1,15 +1,56 @@
+function __fish_brew_get_cmd
+  for c in (commandline -opc)
+    if [ (echo $c | string sub -l 1) != "-" ]
```

Comment:
String takes arguments on the commandline, so `string sub -l 1 -- $c` would work - the `echo` isn't necessary.

Even better is to use `if not string match -q -- '-*' $c`